### PR TITLE
Improve installation instructions

### PR DESCRIPTION
**Pull request by issyl0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,11 @@ Install `go`, `glide`, and `terraform`.
...
-cd terraform-provider-concourse
+git clone https://github.com/alphagov/terraform-provider-concourse $GOPATH/src/github.com/alphagov/
+cd $GOPATH/src/github.com/alphagov/terraform-provider-concourse
 glide install
 go build
-cp terraform-provider-concourse /path/to/my/project
-cd /path/to/my/project
-terraform init
-terraform apply
+cp terraform_provider_concourse ~.terraform.d/plugins/darwin_amd64
```
Comment (Krenair) [inline README.md]:
```suggestion
cp terraform_provider_concourse ~/.terraform.d/plugins/darwin_amd64
```

File: README.md [hunk 2]
```diff
@@ -20,13 +20,11 @@ Install `go`, `glide`, and `terraform`.
 ## How to build and use
 
 ```
-cd terraform-provider-concourse
+git clone https://github.com/alphagov/terraform-provider-concourse $GOPATH/src/github.com/alphagov/
```
Comment (Krenair) [inline README.md]:
```suggestion
git clone https://github.com/alphagov/terraform-provider-concourse $GOPATH/src/github.com/alphagov/terraform-provider-concourse
```

File: README.md [hunk 3]
```diff
@@ -20,13 +20,11 @@ Install `go`, `glide`, and `terraform`.
...
+cd $GOPATH/src/github.com/alphagov/terraform-provider-concourse
 glide install
 go build
-cp terraform-provider-concourse /path/to/my/project
-cd /path/to/my/project
-terraform init
-terraform apply
+cp terraform_provider_concourse ~/.terraform.d/plugins/darwin_amd64
```
Comment (Krenair) [inline README.md]: cp: terraform_provider_concourse: No such file or directory

**Comment by issyl0** [comment]
OK, scratch this. Let me do everything again from the start.

File: README.md (hunk 3 above)
Comment (Krenair) [reply README.md]:
```suggestion
cp terraform-provider-concourse ~/.terraform.d/plugins/darwin_amd64
```

**Comment by issyl0** [comment]
There we go!

**Pull request merged**